tests/InstallCommandTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace RonasIT\Clerk\Tests;
+
+use RonasIT\Support\Traits\MockTrait;
+
+class InstallCommandTest extends TestCase
+{
+    use MockTrait;
+
+    public function testRun()
+    {
+        $authConfigPath = base_path('config/auth.php');
+
+        $this->mockNativeFunction(
+            '\Winter\LaravelConfigWriter',
+            [
+                $this->functionCall('file_exists', [$authConfigPath], true),
+                $this->functionCall('file_get_contents', [$authConfigPath], $this->getFixture('auth.php')),
+                $this->functionCall('file_put_contents', [
+                    $authConfigPath,
+                    $this->getFixture('auth_after_changes.php'),
+                ], $this->getFixture('auth_after_changes.php')),
```

Comment:
any reasons to return new content as the function result? accordingly to the docs it should return the number of bytes which were wrote, let's just set it to `1`